spec/features/apo_spec.rb

```diff
@@ -56,9 +56,7 @@
     expect(page).to have_text 'created'
 
     # Shows a link to the agreement
-    expect(page).to have_link 'Test Agreement',
-                              href: solr_document_path(agreement.externalIdentifier),
-                              wait: 30 # This link is rendered in a turbo-frame, so wait longer than the default
+    expect(page).to have_link href: solr_document_path(agreement.externalIdentifier)
```

Comment:
Has this previous one failed? Can you show me where?

Reply:
Ah, this one: https://app.circleci.com/pipelines/github/sul-dlss/argo/4724/workflows/fa379774-4058-4999-884e-bffdc341ed41/jobs/10332 ?

Reply:
I feel like this just completely removes the behavior I was testing for in the first place.